app/src/main/java/org/cru/godtools/fragment/ToolDetailsFragment.java

```diff
@@ -282,6 +280,7 @@ private void updateViews() {
         bindShares(mShares, mTool);
         if (mDescription != null) {
             mDescription.setText(ModelUtils.getTranslationDescription(getContext(), mLatestTranslation, mTool));
+            Linkify.addLinks(mDescription, Linkify.WEB_URLS);
```

Comment:
This can be accomplished with the `android:autoLink="email|web"` xml attribute. see: https://github.com/CruGlobal/godtools-android/blob/develop/app/src/main/res/values/styles_about.xml#L6